site/content/en/_index.html

```diff
@@ -0,0 +1,42 @@
++++
+title = "Kueue"
+linkTitle = "Kueue"
+
++++
+
+{{< blocks/cover title="Welcome to Kueue" image_anchor="top" height="full" color="orange" >}}
+<div class="mx-auto">
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
+		Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+	</a>
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/kubernetes-sigs/kueue">
+		Github <i class="fab fa-github ml-2 "></i>
+	</a>
+	<p class="lead mt-5">Kubernetes-native Job Queueing</p>
+	{{< blocks/link-down color="info" >}}
+</div>
+{{< /blocks/cover >}}
+
+
+{{% blocks/lead color="primary" %}}
+Kueue is a set of APIs and controller for job queueing.
```

Comment:
Kueue is a set of APIs and controller for job queueing that allows you to build a batch system on  Kubernetes